go/vt/mysqlctl/mysqld.go

```diff
@@ -676,6 +679,32 @@ func (mysqld *Mysqld) Init(ctx context.Context, cnf *Mycnf, initDBSQLFile string
 	return nil
 }
 
+// For debugging purposes show the last few lines of the MySQL error log.
+// Return a suggestion (string) if the file is non regular or can not be opened.
+// This helps prevent cases where the error log is symlinked to /dev/stderr etc,
+// In which case the user can manually open the file.
+func readTailOfMysqldErrorLog(fileName string) string {
+	file, err := os.Open(fileName)
+	buf := make([]byte, maxLogFileSampleSize)
+	if err != nil {
+		return fmt.Sprintf("could not open mysql error log: %s", fileName)
```

Comment:
Log the error message:

```suggestion
		return fmt.Sprintf("could not open mysql error log (%v): %v", fileName, err)
```